python/packages/autogen-ext/src/autogen_ext/models/openai/_openai_client.py

```diff
@@ -630,8 +632,7 @@ async def create(
                 for x in choice.logprobs.content
             ]
 
-        if isinstance(content, str) and self._model_info["family"] == ModelFamily.R1:
```

Comment:
We shouldn't remove this as this is for local R1 models. Instead, we should add a check for `thought is None` and then proceed with parsing.